packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -19,10 +19,16 @@ export const GRID_UNMOUNT = 'unmount';
 export const GRID_ELEMENT_FOCUS_OUT = 'gridFocusOut';
 export const GRID_CELL_CLICK = 'cellClick';
 export const GRID_DOUBLE_CELL_CLICK = 'doubleCellClick';
```

Comment:
The only thing that is now bugging me is this name 😄  everything else is `GRID_CELL_*` can we rename that event to `GRID_CELL_DOUBLE_CLICK` (same for the row double click one)?

Reply:
All done and nice :)